test/system/dashboard_v2_test.rb

```diff
@@ -0,0 +1,22 @@
+require "application_system_test_case"
+
+class DashboardV2Test < ApplicationSystemTestCase
+  Capybara.default_max_wait_time = 60
+
+  test 'view trending topics' do
```

Comment:
This is great and it's awesome to see a system test suite as a testing space for the whole project showing up here!